tests/kafkatest/sanity_checks/test_mirror_maker.py

```diff
@@ -78,13 +99,28 @@ def test_end_to_end(self):
 
         self.producer.start()
         self.producer.wait()
+
+        t0 = time.time()
         self.consumer.start()
-        self.consumer.wait()
+        node = self.consumer.nodes[0]
 
-        num_consumed = len(self.consumer.messages_consumed[1])
+        wait_until(lambda: self.consumer.alive(node),
+            timeout_sec=10, backoff_sec=.2, err_msg="Consumer was too slow to start")
+        self.logger.info("consumer started in %s seconds " % str(time.time() - t0))
+
+        # Verify that log output is happening
```

Comment:
Delete this block - this was a specific check for ensuring log output in the test_console_consumer